src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVariantContextUtils.java

```diff
@@ -1349,4 +1351,119 @@ private static int indexOfSameAllele(final VariantContext vc, final Allele allel
 
         return -1;
     }
+
+    /**
+     * Add chromosome Counds (AC, AN and AF) to the VCF header lines
+     *
+     * @param headerLines the VCF header lines
+     */
+    public static void addChromosomeCounts(final Set<VCFHeaderLine> headerLines) {
+        headerLines.add(VCFStandardHeaderLines.getInfoLine(VCFConstants.ALLELE_COUNT_KEY));
+        headerLines.add(VCFStandardHeaderLines.getInfoLine(VCFConstants.ALLELE_NUMBER_KEY));
+        headerLines.add(VCFStandardHeaderLines.getInfoLine(VCFConstants.ALLELE_FREQUENCY_KEY));
+    }
+
+    /**
+     * Get the number of called alleles for a genotype. Also, increment the number of called alternate alleles.
+     *
+     * @param calledAltAlleles number of called alternate alleles for all genotypes
+     * @param genotype         genotype
+     * @return incremented called alleles
+     * @throws IllegalArgumentException if {@code calledAltAlleles} or {@code genotype} are {@code null}.
+     */
+    protected static int getCalledChromosomeCounts(final Map<Allele, Integer> calledAltAlleles, final Genotype genotype) {
```

Comment:
I think `updateCalledAlleleCounts` may be more clear, and it highlights it that will be modifying the input map.  I still sort of think this should be two different functions.  One that gets the count and one that does the map updating, but if that is to inefficient this is fine.